### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KhrystynaPavlikovska**
- added QC, BA team in Teams
- added one to Dev and BA team

### CODE REVIEW

**Review by abalanovsky — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Review by MariaAfonina — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -292,20 +294,37 @@ Only the release should merge into the main branch!!!
 [![@abalanovsky](https://avatars.githubusercontent.com/u/108689551?v=4)](https://github.com/abalanovsky)
 [![@OlyaKorchan](https://avatars.githubusercontent.com/u/17857767?v=4)](https://github.com/OlyaKorchan)
 [![@MariaAfonina](https://avatars.githubusercontent.com/u/104988390?v=4)](https://github.com/MariaAfonina)
-[![@]()]()
+[![@Marichka0406](https://avatars.githubusercontent.com/u/121502737?v=4)](https://github.com/Marichka0406)
```
Comment (MariaAfonina) [inline README.md:297]: Something wrong with this picture

Screenshot 2023-12-15 at 12 05 13

Comment (KhrystynaPavlikovska) [reply README.md:297]: this bug has happened before. 
It will dissapear after merging i suppose

Comment (MariaAfonina) [reply README.md:297]: Okay, but check this after merging, please.

Comment (KhrystynaPavlikovska) [reply README.md:297]: Deal

**Review by Tolik170 — APPROVED** [review]

**Pull request merged**